new/LIT6185DebatePawli.xml

```diff
@@ -0,0 +1,62 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6185DebatePawli" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ቃል፡ ዘደረሰ፡ አባ፡ ጳውሊ፡ በእንተ፡ ተዋሥኦቱ፡ ምስለ፡ ሰይጣን፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Qāl za-darasa ʾabbā Ṗāwli baʾǝnta
+                tawāśǝʾotu mǝsla Sayṭān</title>
+                <title xml:lang="en" xml:id="t2">Debate of ʾabbā Ṗāwli with Satan</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DE"/>
```

Comment:
MKr

Reply:
Yes, there seems no indication of your name, @MarcinKrawczuk, in this file at all, either as editor or in the change elements